app/styles/_theme.scss

```diff
@@ -437,3 +437,16 @@ label,
     }
   }
 }
+
+// content for screenreader only ==============================================================
+
+.visually-hidden { 
+    position: absolute !important;
+    clip: rect(1px 1px 1px 1px); /* IE6, IE7 */
+    clip: rect(1px, 1px, 1px, 1px);
+    padding:0 !important;
+    border:0 !important;
+    height: 1px !important;
+    width: 1px !important;
+    overflow: hidden;
+} 
```

Comment:
Ne devrait-on pas avoir également la série des `body:hover` ?
(ref : http://a11yproject.com/posts/how-to-hide-content/ )